typescript/packages/reality-capture-modeling/package.json

```diff
@@ -28,19 +28,13 @@
 	},
 	"license": "MIT",
 	"dependencies": {
-		"axios": "^1.6.0"
-	},
-	"peerDependencies": {
-		"@itwin/core-bentley": "^4.0.0",
-		"@itwin/core-common": "^4.0.0"
+		"axios": "^1.6.0",
+		"@itwin/reality-capture-common": "workspace:*",
+		"@itwin/reality-data-client": "workspace:*"
```

Comment:
this is only used for tests, so you can move it to dev dep

Reply:
Fixed